src/Umbraco.Web.UI.Client/src/packages/core/workspace/entity-detail/entity-detail-workspace-base.ts

```diff
@@ -258,25 +268,30 @@ export abstract class UmbEntityDetailWorkspaceContextBase<
 		actionEventContext.dispatchEvent(event);
 	}
 
+	#allowNavigateAway = false;
```

Comment:
Maybe it would be good to make a comment with a description on why this boolean is needed. :-)

Reply:
Good point. I have added a comment 👍